imports/plugins/core/i18n/client/containers/localizationSettings.js

```diff
@@ -135,12 +134,15 @@ function composer(props, onData) {
     value: "",
     label
   }];
-  const timezones = moment.tz.names();
-  for (const timezone of timezones) {
-    timezoneOptions.push({
-      value: timezone,
-      label: timezone
-    });
+  const moment = props.momentTimezone;
+  if (moment) {
+    const timezones = moment.names();
+    for (const timezone of timezones) {
+      timezoneOptions.push({
+        value: timezone,
+        label: timezone
+      });
+    }
```

Comment:
It's worth considering that rather than hit the server and bring down all of moment-timezone library, we could just make a method that returns this array and `Meteor.call` that?